### PR TITLE
Use Write-Host instead of echo

### LINKED ISSUE #16517: make.ps1 fails silently when .NET Core is not available
### Issue Summary
When .NET Core is not installed `make.ps1` fails. Running with an argument (`all`, `clean`) fails silently. Running the script without arguments produces this output:
```
Command list:

  all, a              Builds the game and its development tools.
  dependencies, d     Copies the game's dependencies into the main game folder.
  version, v          Sets the version strings for the default mods to the
                      latest version for the current Git branch.
  clean, c            Removes all built and copied files. Use the 'all' and
                      'dependencies' commands to restore removed files.
  test, t             Tests the default mods for errors.
  check, ck           Checks .cs files for StyleCop violations.
  check-scripts, cs   Checks .lua files for syntax errors.
  docs                Generates the trait and Lua API documentation.

Enter command: all
Press enter to continue.
```
Pressing enter terminates the script. 

#### System Information
- **Operating System:** Windows 7
- **.NET / Mono Version:** .NET 4.7.1
- **OpenRA Version:** bleed

### PR DESCRIPTION
**Pull request by teinarss**
Closes #16517

### CODE REVIEW

**Comment by pchote** [comment]
We use `echo` in a bunch of other places in the script, and i'm pretty sure they work. Why do these ones not?

**Comment by pchote** [comment]
Is there any reason not to use `write-host` to replace all echos?

**Comment by teinarss** [comment]
Not as long as we don't use these outputs in some other place, we don't use
this script in the build system?

**Comment by teinarss** [comment]
Updated

File: make.ps1 [hunk 1]
```diff
@@ -154,7 +154,7 @@ function Check-Scripts-Command
 {
 	if ((Get-Command "luac.exe" -ErrorAction SilentlyContinue) -ne $null)
 	{
-		echo "Testing Lua scripts..."
+		Write-Host "Testing Lua scripts..."  -ForegroundColor Blue
```
Comment (abcdefg30) [inline make.ps1]: Double blank space here and in several other places.

Comment (teinarss) [reply make.ps1]: Updated

**Comment by reaperrr** [comment]
Not tested in more detail yet, but

```
[00:37] <+reaperrr> btw pchote, I forgot where [RequireExplicitImplementation] would trigger if something is not explicit, was it during compile, check or test?
[00:38] <+reaperrr> could that have regressed due to the make/csproj refactor?
[00:38] <+pchote> its the --check-explicit-interfaces utility rule, which is run in check
[00:38] <+pchote> and possibly
[00:38] <+pchote> but more likely related to #16529
```

might also be related.

**Review by abcdefg30 — DISMISSED** [review]
Looks good to me.

**Comment by reaperrr** [comment]
One nit: Could we use Cyan instead of Blue? On my screen Blue is dark enough that it's rather hard to read against the black background.
👍 after that.

**Comment by teinarss** [comment]
Updadet

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/621054f2fb36e07a9c6c90d32b8e2312add94eef...31f09df14792dd22ec33b1050e1eb1a0606a722e

**Pull request merged**